angular/src/app/modules/delivery-management/delivery/available-resource-tab/will-pool/will-pool.component.ts

```
@@ -0,0 +1,386 @@
+  public isExpands: { [id: number]: boolean } = {};
+  public numberDataRow: number = 3;
+  public showIconExpandCollapeAll: boolean;
+  public isExpandAll: boolean = false;
+  // edit note
+  public isEditNote: { [id: number]: boolean } = {};
+  public originalNoteValue: string = "";
+  // search username
+  public searchText: string = "";
+  public oldSearchText: string = "";
+
+  protected list(_request: PagedRequestDto, pageNumber: number, _finishedCallback: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)